packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -1127,6 +1127,7 @@ export default class CashierStore extends BaseStore {
         }
 
         this.config.account_transfer.selected_from = selected_from;
+        this.setTransferFee();
```

Comment:
transfer fee function is using both the from and the to currencies to determine what value to show. should we keep it in both onChangeTransferFrom and onChangeTransferTo functions instead of keeping it in one of them?

Reply:
Actually, because of the logic behind the onChangeTransferTo function, it's work is based on onChangeTransferFrom function, and by the time when onChangeTransferTo finishes it's job, the `this.config.account_transfer.selected_from` value is not updated to a new one yet, therefore `setTransferFee()` function will get the wrong inputs and the result will always be incorrect.